wrapanapi/msazure.py

```diff
@@ -268,6 +270,27 @@ def list_subscriptions(self):
                 self.subscription_client.subscriptions.list() if
                 s.state == SubscriptionState.enabled]
 
+    def switch_subscription(self, new_subscription):
```

Comment:
With my implementation of `__setattr__` in #205 this method may not be needed anymore. Doesn't hurt to have it, for an explicit call to change subscriptions.